server/rest/src/main/java/org/infinispan/rest/stream/ExtendedChunkedInput.java

```diff
@@ -0,0 +1,18 @@
+package org.infinispan.rest.stream;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.stream.ChunkedInput;
+import io.netty.handler.stream.ChunkedWriteHandler;
+
+public interface ExtendedChunkedInput<T> extends ChunkedInput<T> {
```

Comment:
We can delete this one.

Reply:
Oh weird, thought I had. Removing.

Reply:
Oh, I guess I didn't push the most up to date code I had, sorry about that.